osf/models/metaschema.py

```diff
@@ -29,29 +32,38 @@
 
 
 class AbstractSchemaManager(models.Manager):
-    def get_latest_version(self, name, only_active=True):
+
+    def get_latest_version(self, name):
         """
         Return the latest version of the given schema
         :param str only_active: Only returns the latest active schema
         :return schema
         """
-        schemas = self.filter(name=name, active=True) if only_active else self.filter(name=name)
-        sorted_schemas = schemas.order_by('schema_version')
-        if sorted_schemas:
-            return sorted_schemas.last()
-        else:
-            return None
+        return self.filter(name=name).order_by('schema_version').last()
 
-    def get_latest_versions(self, only_active=True):
+    def get_latest_versions(self):
         """
         Returns a queryset of the latest version of each schema
-        :param str only_active: Only return active schemas
         :return queryset
         """
-        latest_schemas = self.filter(visible=True)
-        if only_active:
-            latest_schemas = latest_schemas.filter(active=True)
-        return latest_schemas.order_by('name', '-schema_version').distinct('name')
+        return self.filter(visible=True).order_by('name', '-schema_version').distinct('name')
+
+    def get_latest_versions_and_allow_egap_admins(self, request):
```

Comment:
I tend to agree with @UdayVarkhedkar here; having `get_latest_version` call an `allow_egap_admins` function would make for a smaller diff; right now, [there](https://github.com/CenterForOpenScience/osf.io/pull/9282/files#diff-ed5e8962a2d66ef00f0e19950ebaced2R37) are [multiple](https://github.com/CenterForOpenScience/osf.io/pull/9282/files#diff-ce6f58567740c2f9e6e012108003c941R46) [places](https://github.com/CenterForOpenScience/osf.io/pull/9282/files#diff-ce6f58567740c2f9e6e012108003c941R65) in the codebase that had to be changed (and will have to be changed in the future, more importantly) to accommodate this particular method.